core/src/main/java/hudson/TcpSlaveAgentListener.java

```diff
@@ -228,9 +232,9 @@ public void shutdown() {
 
     private final class ConnectionHandler extends Thread {
         private static final String DEFAULT_RESPONSE_404 = "HTTP/1.0 404 Not Found\r\n" +
-                        "Content-Type: text/plain;charset=UTF-8\r\n" +
-                        "\r\n" +
-                        "Not Found\r\n";
+                "Content-Type: text/plain;charset=UTF-8\r\n" +
```

Comment:
please try and prevent whitespace changes in unrelated code.

Reply:
It was produced from `mvn: spotless apply`. Will cross check before commit next time onwards